pkg/vm/vm.go

```diff
@@ -1302,9 +1332,18 @@ func (v *VM) jumpIf(ctx *Context, offset int, cond bool) {
 
 // getJumpOffset returns instruction number in a current context
 // to a which JMP should be performed.
-// parameter is interpreted as little-endian int16.
+// parameter should have length either 1 or 4 and
+// is interpreted as little-endian.
 func (v *VM) getJumpOffset(ctx *Context, parameter []byte, mod int) int {
-	rOffset := int16(binary.LittleEndian.Uint16(parameter))
+	var rOffset int32
```

Comment:
I think we can declare it an `int` and avoid `int32 -> int` conversion because our `int` is guaranteed to be at least 32 bits wide.

Reply:
It will be needed when converting from `uint32` then, because `int` can be 64-bytes long, so I see no difference.

Reply:
True, the number of conversions is gonna be the same.